src/System.Diagnostics.TextWriterTraceListener/tests/DelimiterWriteMethodTests.cs

```diff
@@ -8,14 +8,14 @@
 
 namespace System.Diagnostics.TextWriterTraceListenerTests
 {
-    public class DelimiterWriteMethodTests : IDisposable
+    public class DelimiterWriteMethodTests : FileCleanupTestBase
     {
         private readonly Stream _stream;
         private readonly string _fileName;
 
         public DelimiterWriteMethodTests()
         {
-            _fileName = string.Format("{0}.xml", GetType().Name);
+            _fileName = $"{GetTestFilePath(null, GetType().Name)}.xml";
```

Comment:
I did it because it was from the constructor but didn't quite think that GetType().Name will return the same as [CallerMemberName] as the constructor is the caller. Can't believe I didn't think on that :( thanks for pointing it out!